packages/app/src/messages/components/messages/messages.component.tsx

```diff
@@ -129,10 +132,11 @@ const Messages: FunctionComponent<MessagesProps> = ({
   const [draftMessage, setDraftMessage] = useState<Message>()
   const [content, setContent] = useState("")
   const debouncedContent = useDebounce(content, 1000)
-  const { selectedRows, allRowsSelected, toggleAll, resetRows, ...rest } =
-    useTableSelect<Thread>(threads)
+  // const { selectedRows, allRowsSelected, toggleAll, resetRows, ...rest } =
```

Comment:
let's remove it

Reply:
Removed in [106999b](https://github.com/mudita/mudita-center/pull/864/commits/106999bfe9a7c55f9033a87e93120de5f3ffffd0)